File: 프로그래머스/unrated/181925. 수 조작하기 2/수 조작하기 2.js
function solution(numLog) {
    let result = '';
    let memo = numLog[0];
    
    for(let i of numLog) {
        if(i === memo+1) {
            result += 'w'
            memo = i;
        }else if(i === memo-1) {
            result += 's'
            memo = i
        }else if(i === memo+10) {
            result += 'd'
            memo = i
        }else if(i === memo-10){
            result += 'a'
            memo = i
        }
    }
    return result
}

File: 프로그래머스/unrated/181925. 수 조작하기 2/수 조작하기 2.py
def solution(numLog):
    result = '';
    memo = numLog[0]
    
    for n in numLog:
        if n == memo + 1:
            result += 'w'
            memo = n;
        elif n == memo - 1:
            result += 's'
            memo = n;
        elif n == memo + 10:
            result += 'd'
            memo = n;
        elif n == memo - 10:
            result += 'a'
            memo = n;
    
    return result
        
    